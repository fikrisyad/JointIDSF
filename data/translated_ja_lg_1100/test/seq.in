プロジェクト「ゴスの新しい顔」を見つけてください。
このアーティストを「Spotlight on Country 2016」プレイリストに追加する
Mac n Cheeseのプレイリストに曲を追加する。
このアルバムをオールドスクールデスメタルに追加する
テレビドラマ「ジャージー・ボーイズ」を探す
ユタ州の近くでは、午前3時頃、どのくらい寒くなるのでしょうか。
アニメ映画
ピーター・マーフィーの80年代の良いEPを再生する。
♪Can you find me hey man
ガンビアのユダヤ料理レストランを予約したい。
は、最寄りの映画館で正午からレアバースを上映しています。
は、カウボーイ・カンティーン（Cowboy canteen）の演奏です。
カリフォルニア州アーバンクのジェームス・D・コンレイ・ハウスでテーブルを貸してほしい。
最寄りの映画館で今上映中の作品
最寄りの映画館で上映される作品を探す
オーランドのアウトドアカフェを6名様でご予約ください。
本書の評価：6点満点中4点
我が家の仕事帰りのプレイリストにテーマソングができました。
青春を演じる映画館はあるのか
エメラド・サン・ピエール・ミクロンの9月26日の予報は？
近所にフィルムが欲しい
しろくろをつける
married to the enemy 2 を映画館で観たい。
ディプソンシアターで開催される「ミス・ビショップへの応援歌」の上映時間をお知らせします。
エピック・コンディション・ピクチャーの場所を教えてください。
ランドリープレイリストに40calの曲を追加する。
sarah sleanをプレイリストに追加 mellowed out gaming
アウトキャストが見たい
リーガル・エンタテインメント・グループの映画スケジュールを検索する
farhad daryaの曲をviral evergreensに追加する。
人気レストランを5名で予約
セクシュアリティーショー論について、3つのエッセイをご覧ください。
最寄りの映画館で「アウター・スペース・コネクション」が上映されるのはいつですか？
indie hipsterに曲を追加する
カウンスバーグの同じ地域にあるプール付きのオイスターバーを10人分予約しておく。
ホット50プレイリストにアルバムを追加する
はAMCシアターで上映中です。
恐怖の部屋を見てみたい
夜10時ごろの天気予報を知らせてくれ、ここから遠く離れたパーク・ナロドウィ・ブリムストーンヒル要塞で。
jazzy dinnerに楽曲を追加する
6つ星のうち5つを「表現力豊かな加工」に
星を2つ付ける
は、ゼネラル・シネマ・コーポレーションで上映中の「the eye - infinity」です。
ライブ名入り写真を見たい方はこちら
グルーブシャークでJペロ弾けるか？
インマンの近くのレストランを5名で予約したいのですが。
アルバムを私のフロー・エスパニョール・プレイリストに追加する
ザック・スターキーのEPを再生する
12月5日、リフラのモニュメントの天気はどうでしょうか？
Black Sabbath the dio years というプレイリストに Last of Ghetto astronauts を追加してくれませんか？
近作映画
反日ゲリラの回想録は、6点満点中0点に値する。
ジョーダン・ルーデスを「子供向けの遊び」のプレイリストに加えたい。
チャドの天気はどうなるのか？
四曲を探す
プライムミニスターズ：パイオニアはどこで見ることができますか？
Joy Niloの音楽を演奏する
ご近所アニメの上映時間
大脱走劇場の上映スケジュールは？
上映時間
近作映画
70年代のサウンドトラックを聴きたい
テレビ番組「古代イギリスの国歌」を探しています。
グッドリッチ・クオリティ・シアターズで上映スケジュールを確認する
昼に上映している窓際の映画館はどこが一番近いか
私のプレイリストに「hands up」という曲を追加
地域の映画上映スケジュールを検索する
四分六厘
ヘンダーソン・キッズ・サーガをお願いします。
Caryl & Marilyn: Real Friendsに関連する創作物を探してください。
火中の栗を拾う
近所の映画館を紹介
現在の教科書に6点満点中1点をつける
ジンジャーブレッドマンの時間帯は？
上映時間
2025年1月2日、アメリカ・ジョージア州の天気は？
ワークアウトのプレイリストには、もっとクリス・クロスが必要だ。
さがす
シグネ・アンダーソンの最新チャントミュージックを再生します。
ポケモンザ・ムービー2000をさがしています。
どんな映画が観られるか
ユナイテッドパラマウントの映画館で上映する。
キッシュが食べられるレストランをFMで探す
フォレストアブのレストランで今から2 1秒後に予約する。
Show me the courts of chaos
ぼる
即興演奏と題されたこの本を5つ星に評価する
ジュリフあたりのバーを 3 人で予約してくれ ブカティーニを出す店だ
クイーン・オブ・オルガンを表示させることができますか？
午前11時からのアニメの上映スケジュールを教えてください。
マサチューセッツ州は荒れ模様でしょうか
Mr.フューリーの目に映る率 0 of 6
ビート・ザ・デビルがマンシアターで上映されるのは何時ですか？
アルマゲドンサマーというテレビドラマを探す
ゲームを見つける just dance greatest hits
近隣の映画館での上映時間を教えてください。
Play the song memories are my only witness
このアルバムは6点満点で2点です。
Leah Kauffmanをuncharted 4 nathan drakeのプレイリストに追加する。
アブデラマン アブデリの「不可能はない」のようなものはありますか？
40年代のarifの音楽を演奏する
シックスティーズ
ミッシェル・ヒートンを加えて、チョピンです。
サウスカロライナの同じ地域の日没時の天候を表示する
私のパンク必需品プレイリストに、no mystery を入れてください。
フィッシュエクスプレスで予約を探す
フォックスシアターの上映時間を教えてください。
ケリー・ジーンと私のために、エレベンシーのパブを予約してください。
悪の枢軸とその他の刺激的な国家の料理を6点満点で評価する。
PAX WARRIORというサウンドトラックを探す
グジェールのあるレストランを9人で予約する。
ワイオミングヒルズは暑いのか？
ベンガル料理が食べられるタベルナを5人で予約する。
作品を探しています：Nikki
メローバーのプレイリストに曲を追加
複製品を探す: ソングス オブ ザ ヒューマン リーグ
シークレット・サンシャインのマン・シアターでの上映時間は？
呂律が回らない
れきしを4とする
DOGGY DAY SCHOOLのアルバムを探す
私のスペイン・トップ50プレイリストに曲を追加してくれませんか？
ダウンタウンに近いシュラスコ料理を出す店に4人連れて行きたい。
アイマックスコーポレーションの映画スケジュールを教えてください。
ハーストビルの天気は？
秘密の水の評価は4点です。
カナディアン・マウントの危険性を示す
最寄りの映画館で「世界で一番幸せな女の子」を上映しているところを教えてください。
クロニクル・テン・フロム・トゥモローを2点評価
ゲタを履く
とびおりる
crate diggers anonymousにblurred crusadeを追加。
クロニクル ゾンビのウラナミさん 3点
ポリティクスネイションをアル・シャープトンで探す
このアルバムの評価 6点満点中0点
ボーレガードの天気
サーガのマイケル・クレイトンを読みたい。
現在の教科書を6つ星で評価する
ウソツキを五つに評価する
ウーマンオブザブルースに翔翔の曲を入れる
イアン・マクドナルドの1991年のサウンドトラックを再生する。
ショウ・ミー・ムービー・スケジュール
TKK フライドチキン（スリランカ）を3名様でご予約ください。
ジョディ・ウィリアムズのサウンドトラックが聴きたい
back when i knew it all アルバムを探してください。
ニードルパークでパニックを見つける
この秋、アゼルバイジャン周辺で一流のステーキハウスを1名で予約したいのですが。
すべてのアカペラにウィングトラックを追加
夜の通勤プレイリストにこのアーティストを入れて欲しい。
ロックンロールを奏でる
私はこの本の英語辞書を4評価します。
