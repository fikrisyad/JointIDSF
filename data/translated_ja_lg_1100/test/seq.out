O O O O O Obj-Intent O O O O O O
O Obj-Intent O O B-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT O O O O O O
O O O O O O O Obj-Intent O O O O
O Obj-Intent O O O O O
O Obj-Intent O O O O O O O
B-GPE I-GPE O O O O O B-TIME I-TIME I-TIME O O O O O O O O O O
O O
B-PERSON I-PERSON I-PERSON O B-DATE I-DATE I-DATE O O Obj-Intent O O O O
O O O O O O
B-GPE O B-NORP O Obj-Intent O O O O O
O O O O O O O B-TIME O B-ORG O O O O O O O
O O O O O O O O O O Obj-Intent O O
B-GPE I-GPE I-GPE O B-PERSON I-PERSON I-PERSON I-PERSON I-PERSON I-PERSON I-PERSON O Obj-Intent O O O O O
O O O O O O O O O Obj-Intent
O O O O O O O O Obj-Intent O O
B-GPE O O Obj-Intent O B-QUANTITY I-QUANTITY O O Obj-Intent O O O
O O O O B-QUANTITY I-QUANTITY O O B-QUANTITY I-QUANTITY
O O O O O O O O O O O O O O O
O O B-EVENT I-EVENT I-EVENT O O O O
B-ORG I-ORG I-ORG I-ORG I-ORG I-ORG I-ORG O B-DATE I-DATE I-DATE I-DATE O O O O
O O O O O
Obj-Intent O O
O O O O B-ORDINAL O O O O O O O
O O O O O O B-WORK_OF_ART I-WORK_OF_ART I-WORK_OF_ART O O O O O O O Obj-Intent O Obj-Intent O O O O
O O O O O O Obj-Intent O O O O O
O O O O B-QUANTITY I-QUANTITY O Obj-Intent O O O O
O O O O O O O B-EVENT I-EVENT Obj-Intent
O O O O O
B-ORG I-ORG I-ORG I-ORG I-ORG O O Obj-Intent O O O
B-PRODUCT I-PRODUCT O Obj-Intent O O O O O O O
O O O B-QUANTITY I-QUANTITY O O
B-PRODUCT I-PRODUCT I-PRODUCT O O O O B-QUANTITY I-QUANTITY O O O Obj-Intent O O
O O O O O O O O O O O O O O O O O O O O O O
B-PRODUCT I-PRODUCT O Obj-Intent O O O
B-ORG O O O O O O O O B-ORG O B-QUANTITY I-QUANTITY O O O O O O
O B-QUANTITY I-QUANTITY O O Obj-Intent O O O
O O O O O Obj-Intent O O
O O Obj-Intent O O O O O
B-TIME I-TIME I-TIME O O O O O O O O O O O O O O O O O O O O Obj-Intent O O
O O O Obj-Intent O O O
B-QUANTITY I-QUANTITY O O O B-QUANTITY I-QUANTITY O O O O O O O O O
Obj-Intent O B-QUANTITY I-QUANTITY O
O O B-ORG I-ORG I-ORG I-ORG I-ORG O O O O O B-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT O O O
O O O O O O O O O
O B-ORG O B-PERSON O O O O
B-PERSON O O O Obj-Intent O B-QUANTITY I-QUANTITY O O O O O O O O
Obj-Intent O O O O O B-ORG O O O O O O
B-PERSON I-PERSON I-PERSON O Obj-Intent O O O
B-DATE I-DATE I-DATE I-DATE O B-FAC O O O O O O O O O
B-PRODUCT I-PRODUCT I-PRODUCT B-PERSON I-PERSON O O O O O B-ORG I-ORG I-ORG I-ORG O O O O O O O O O
O O
O O O O O O O B-QUANTITY I-QUANTITY O O B-QUANTITY I-QUANTITY O O O O
B-PERSON I-PERSON I-PERSON O O O O O O O O O O O O O O
O O O O O O O O O
B-QUANTITY I-QUANTITY O O
O O O O O O O Obj-Intent O O O O O
B-PRODUCT I-PRODUCT O Obj-Intent O O O
O O O O O O
B-ORG I-ORG I-ORG O O O O O
O O
O O
B-DATE I-DATE I-DATE O O Obj-Intent O O O
O Obj-Intent O B-DATE B-GPE O O O O O O O O O
O O O O O O O O Obj-Intent O O O
O O O O O O O O O O O O O B-ORDINAL O O
O O O O O O O O O O O O O O
O O O O Obj-Intent O O O
B-TIME I-TIME I-TIME I-TIME
B-PERSON I-PERSON I-PERSON I-PERSON I-PERSON O Obj-Intent O O O O
B-ORG I-ORG I-ORG O B-ORG I-ORG O O O O Obj-Intent O O O O O
O O Obj-Intent O O
O O O O O O
O O O O O B-QUANTITY I-QUANTITY O O B-QUANTITY I-QUANTITY O O
B-PRODUCT O O O O O
O O
B-DATE I-DATE I-DATE I-DATE I-DATE I-DATE O B-GPE O B-GPE I-GPE O O O O
O O O O O O O O O B-PERSON I-PERSON I-PERSON O O O O
O
B-PERSON I-PERSON I-PERSON O O Obj-Intent O O O O O
O O O Obj-Intent O O O O O O
O O O O O O
B-ORG I-ORG O O O O O O O
O O O O Obj-Intent O O O O
O O O O O O B-ORDINAL I-ORDINAL I-ORDINAL I-ORDINAL O O O O
B-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT
O
O O O O O O O Obj-Intent O B-QUANTITY I-QUANTITY O O O O
B-GPE O O O O B-QUANTITY I-QUANTITY O O O O O O O O Obj-Intent O
O O O O Obj-Intent O O O O Obj-Intent O O O O O
B-TIME I-TIME I-TIME O O O O O Obj-Intent O O O O O
B-GPE I-GPE O O Obj-Intent O O
O O O O O O O B-DATE I-DATE I-DATE
O O O O O O B-GPE O O O O O O O O O O
B-ORG I-ORG O O O Obj-Intent O O
O O O O O O O
O O O O O O O Obj-Intent O O O O O
B-PRODUCT I-PRODUCT I-PRODUCT O O O O Obj-Intent
O O O B-QUANTITY I-QUANTITY O O B-QUANTITY I-QUANTITY O O
B-PRODUCT I-PRODUCT O O B-GPE I-GPE I-GPE O O O O O O O
B-ORG O O O O O O O O O O O O O O O O O
B-DATE I-DATE I-DATE O O O Obj-Intent O O O
O
B-PERSON I-PERSON I-PERSON O O O O Obj-Intent O O
B-GPE O O O O O O O Obj-Intent O O O
O O O O O O O O O O Obj-Intent O O O O O
B-ORG I-ORG O Obj-Intent O O
B-ORG I-ORG O O Obj-Intent O O O O O
B-PERSON I-PERSON I-PERSON O O O O O B-GPE O Obj-Intent O O O O O O
O O O O O O O O O O O O Obj-Intent O B-QUANTITY I-QUANTITY O O O O O
B-ORG I-ORG O O O Obj-Intent O O
O O O Obj-Intent O B-QUANTITY I-QUANTITY O O O O
B-FAC I-FAC O O O O O
B-NORP O O O O Obj-Intent O B-QUANTITY I-QUANTITY O O O O
O O O O O O O Obj-Intent
O O O O O O O O
O O O O O O O O O O
B-LOC I-LOC I-LOC O O O O O O O O O O
B-PERSON O O O
Obj-Intent O B-QUANTITY O O
B-ORG I-ORG I-ORG O Obj-Intent O O
O O B-ORG I-ORG I-ORG I-ORG I-ORG I-ORG O Obj-Intent O O O O O O O O O
O O O B-NORP O O O O O B-QUANTITY I-QUANTITY O O O O O
O O O O Obj-Intent O O O O O
O O O O O
O O O O O O B-QUANTITY I-QUANTITY O O
O O O O O Obj-Intent O O
O O O O O O O O B-ORDINAL O O O O O O O O O Obj-Intent O O O O O
B-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT O B-QUANTITY I-QUANTITY Obj-Intent
Obj-Intent O O
O O
O O O O O O O Obj-Intent O
B-WORK_OF_ART I-WORK_OF_ART O B-PERSON O B-QUANTITY I-QUANTITY
B-WORK_OF_ART I-WORK_OF_ART O B-PERSON I-PERSON I-PERSON O O
O O O O B-QUANTITY I-QUANTITY O O B-QUANTITY I-QUANTITY
B-GPE O Obj-Intent
O O B-PERSON I-PERSON I-PERSON O O O O
O O O Obj-Intent O B-QUANTITY I-QUANTITY O O O O
B-GPE O B-QUANTITY I-QUANTITY O O O
B-GPE O B-PERSON I-PERSON O Obj-Intent O O
B-PERSON I-PERSON I-PERSON O B-DATE I-DATE O O Obj-Intent O O O O
O O O O O O Obj-Intent
B-PRODUCT O O O B-GPE Obj-Intent O B-QUANTITY I-QUANTITY O O Obj-Intent O O O
B-PERSON I-PERSON I-PERSON O O O O O O
B-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT I-PRODUCT Obj-Intent O O O O O
O O O Obj-Intent O O
O O O B-GPE O O O O O Obj-Intent O B-QUANTITY I-QUANTITY O O O O O O O O
O O O O O O O O
O O O O O O O Obj-Intent O O O O O
Obj-Intent O O
O O O O O B-LANGUAGE O O B-QUANTITY I-QUANTITY O O O
